From 6294eb4aefa4e815c16183cce776cf6c4c505230 Mon Sep 17 00:00:00 2001
From: Joe Zheng <joe.zheng@intel.com>
Date: Wed, 24 Jul 2019 15:34:36 +0800
Subject: [PATCH 1/2] FIXME: fail to boot with -e option

add uinput device into white list to fix the issue:
F DEBUG : Abort message: 'jni_internal.cc:616] JNI FatalError called:
(system_server) Not whitelisted : /dev/uinput'

need to find out the root cause and implement into BSP

Change-Id: I56ab4d91c94937687a9a22588479b7821c512b8b
Tracked-On: ACP-615
---
 core/jni/fd_utils.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/jni/fd_utils.cpp b/core/jni/fd_utils.cpp
index c5904e0e9e5..aecada369a5 100644
--- a/core/jni/fd_utils.cpp
+++ b/core/jni/fd_utils.cpp
@@ -42,6 +42,7 @@ static const char* kPathWhitelist[] = {
   "/dev/urandom",
   "/dev/ion",
   "/dev/dri/renderD129", // Fixes b/31172436
+  "/dev/uinput",
 };
 
 static const char kFdPath[] = "/proc/self/fd";
-- 
2.21.0

